tests/foreman/api/test_syncplan.py

```diff
@@ -295,10 +319,21 @@ def test_remove_one_product_to_sync_plan(self):
         @Feature: Sync Plan
 
         """
+        syncplan_id = entities.SyncPlan(
+            organization=self.org_id
+        ).create_json()['id']
+        product_id = entities.Product(
```

Comment:
I've significantly updated this test. Let me know if you have any further thoughts on it. Good catch, by the way. :smile: